app/models/user.rb

```diff
@@ -1,11 +1,18 @@
 class User < ActiveRecord::Base
   authenticates_with_sorcery!
 
-  validates :password, length: { minimum: 3 }
-  validates :password, confirmation: true
-  validates :password_confirmation, presence: true
+  validates  :password, length: { minimum: 3 }
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.